File: spot-the-difference/spotTheDifference.js
const spot = (string1, string2) => {
    const out = [];
    string1.split('').forEach((element, index) => {
        if(element !== string2[index]) out.push(index);
    });

    return out;
};

module.exports = { spot };

//https://www.codewars.com/kata/5881460c780e0dd207000084/train/javascript

File: python-venv/spot_the_dffierence/kata_function/spot_the_difference.py
def spot(string1, string2):
    out = []
    split = list(string1)

    for i in range(len(split)):
        if(split[i] != string2[i]):
            out.append(i)

    return out

#https://www.codewars.com/kata/5881460c780e0dd207000084/train/python